main.c

```diff
@@ -383,11 +383,13 @@ inline uint8_t min(uint8_t a, uint8_t b) { return a <= b ? a : b; }
  * Update lighting table as per animation
  */
 static inline void animationCallback() {
-
   // If the foreground is set we skip the animation as a way to avoid it
   // overrides the foreground
-  if (!is_foregroundColor_set &&
-      profiles[currentProfile].animationSpeed[currentSpeed] > 0) {
+  if(is_foregroundColor_set){
```

Comment:
Space between ) and {

Reply:
Nice. I didn't know about make clang-format-ci until now, but I think it's a welcome addition.

It's passing now and I've pushed it to my branch. Do I need to do anything more?